src/main/java/frc/robot/RobotContainer.java

```diff
@@ -27,8 +28,9 @@ public RobotContainer() {
 	private void mapButtonBindings() {
 		Robot.logger.addInfo("RobotContainer", "Start to map button bindings");
 
-		// Toggle Intake
+		// Intake
 		x_button.whenPressed(new ToggleIntake());
+		l_bump.whenPressed(new ReverseIntake());
```

Comment:
Remove this from controls for now, it's won't be used individually most likely.
b_button will be used for a timed reverse (ball gets jammed; timed command)
left_menu will be used for emptying the entire system of balls (reverse feeder, reverse conveyor, reverse intake; needs to be a command group)